CODE_OF_CONDUCT.md

```diff
@@ -1,7 +1,8 @@
 <!--
-Copyright (C) 2024 Arm Limited or its affiliates and Contributors. All rights reserved.
+Copyright (C) 2025 Arm Limited or its affiliates and Contributors. All rights reserved.
```

Comment:
Should be `2024 - 2025`

Reply:
Why is that? I was thinking of the usual convention of having the current year alongside the copyright logo.

Reply:
Checked, should be latest year.